common/utils/jwt.go

```diff
@@ -12,6 +12,22 @@ func GenerateSignedToken(secret []byte, data jwt.Claims) (string, error) {
 	return token.SignedString(secret)
 }
 
+func FetchIdFromSignedUserToken(secret string, signed_token string) (string, error) {
```

Comment:
I agree, but the main reason of the duping is to allow for `userId` to be directly casted to a string rather than to a `[]string`. I'll put this into an issue.

While writing this comment out I realize that this line:

https://github.com/HackIllinois/api/blob/f66cf00e69f58a1efa43cfd85db67dfbe44cf3b2/common/utils/jwt.go#L28

does not have any error checking and will cause a panic in case some token does not parse correctly.